FirebaseSessions/Sources/FirebaseSessions.swift

```diff
@@ -19,6 +19,12 @@ import FirebaseInstallations
 
 // Avoids exposing internal FirebaseCore APIs to Swift users.
 @_implementationOnly import FirebaseCoreExtension
+@_implementationOnly import GoogleDataTransport
+
+private enum GoogleDataTransportConfig {
+  static let sessionsMappingFileID = "1974"
```

Comment:
Is this the log source? If so, can we call it a LOG SOURCE instead of calling it Mapping File ID?

Reply:
I'm down to switch it. They call it mapping file ID in the GDT API so that's why I went with that

Reply:
Also, this was confusing to me, but constants are lower camel case in the style guide: go/swift-style#global-constants